dockers/llm.rag.service/serveragllm.py

```diff
@@ -14,22 +14,90 @@
 
 ########
 # Setup model name and query template parameters
-model = "mosaicml/mpt-7b-chat"
+MICROSOFT_MODEL_ID="microsoft/Phi-3-mini-4k-instruct"
+MOSAICML_MODEL_ID = "mosaicml/mpt-7b-chat"
+RELEVANT_DOCS_DEFAULT = 2
+MAX_TOKENS_DEFAULT = 64
+MODEL_TEMPERATURE_DEFAULT = 0.01
+MODEL_ID_DEFAULT = MOSAICML_MODEL_ID
+
 template = """Answer the question based only on the following context:
 {context}
 
 Question: {question}
 """
 os.environ["TOKENIZERS_PARALLELISM"] = "false"
 
+def ensure_integer(value, name):
```

Comment:
This method doesn't seem to be used.

Reply:
Good catch - this was an original method I tried unsuccessfully to move from the string-based Docker env vars to int/float values to be usable by the langchain methods. I will remove it.

Reply:
Removed with the most recent push.